From fb71007d6c7e5c2d8df4fcff0cab91e48620c48a Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Mon, 4 Jun 2018 16:46:01 -0700
Subject: [PATCH 0534/1200] drivers: usb eth: display interface name

Display ethernet interfaces detected over usb while executing
usb start/reset as there is no way to know about them using commands.

Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 drivers/usb/eth/usb_ether.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/eth/usb_ether.c b/drivers/usb/eth/usb_ether.c
index 36734e2e51..66680b4596 100644
--- a/drivers/usb/eth/usb_ether.c
+++ b/drivers/usb/eth/usb_ether.c
@@ -241,6 +241,7 @@ static void probe_valid_drivers(struct usb_device *dev)
 			if (eth_write_hwaddr(eth, "usbeth",
 					usb_max_eth_dev - 1))
 				puts("Warning: failed to set MAC address\n");
+			printf("	%s", eth->name);
 			break;
 			}
 		}
-- 
2.22.0

